test/test.pl

```diff
@@ -1,30 +1,6 @@
-#!/usr/bin/env swipl
-/** <module>  Unit tests for the tokenize library
- *
- * To run these tests, execute this file
- *
- *    ./test/test.pl
- */
-
-:- initialization(main, main).
-
-main(_Argv) :-
-    run_tests.
-
+:- use_module(prolog/tokenize).
 :- begin_tests(tokenize).
 
-:- dynamic user:file_search_path/2.
```

Comment:
Doesn't this part still have value, in letting the module be loaded from other locations? Does it interfere with the other changes?

Reply:
I believe that that is not necessary but better ask @Anniepoo :)

Reply:
What happens if you try to load this file while in the `test.pl` dir?

Reply:
I get

```
# sf @ comp in ~/oss/tokenize/test on git:dda25bc o [16:01:44]
$ swipl test.pl
ERROR: /Users/sf/oss/tokenize/test/test.pl:1:
	source_sink `prolog/tokenize' does not exist
Warning: /Users/sf/oss/tokenize/test/test.pl:1:
	Goal (directive) failed: user:use_module(prolog/tokenize)
```

Reply:
if this doesn't hurt anything, I'd rather leave this part in so we get the flexibility.

Reply:
Yeah, you are right @shonfeder, fixed it!